201-web-app-asev2-create/azuredeploy.json

```diff
@@ -10,48 +10,39 @@
         },
         "aseLocation": { 
             "type": "string",
-            "defaultValue": "South Central US",
             "metadata": { 
                 "description": "Location of the App Service Environment" 
             } 
-        }, 
-        "existingVirtualNetworkName": {
+         },
+        "existingSubnetResourceId": {

```

Comment:
revert this change - pass in the name of the vnet and the rg.  The resourceId can be constructed from that

Reply:
I will look into how to construct a RID in a template.  Would need the subnet name also

Reply:
Ok, I realized I can't do this.  I must take the RID of the subnet.  Many of our customers create their ASEs that are not just in a different resource group than the VNet but also they are in different subscriptions.  Collecting the subscription ID, resource group name, VNet name, and subnet name as separate items is a poor substitute for copy/paste of the subnet RID in the first place.

Reply:
you can add a defaultValue for each using the functions for retrieving the values - make the common case simple.

Reply:
this still needs to be done